op-node/rollup/driver/state.go

```diff
@@ -334,6 +339,10 @@ func (s *Driver) eventLoop() {
 			} else if !bytes.Equal(unsafeHead[:], resp.hash[:]) {
 				resp.err <- fmt.Errorf("block hash does not match: head %s, received %s", unsafeHead.String(), resp.hash.String())
 			} else {
+				if err := s.sequencerNotifs.SequencerStarted(); err != nil {
+					resp.err <- fmt.Errorf("sequencer start notification: %w", err)
+					continue
+				}
```

Comment:
I'd suggest always persisting it when the signal is received, even if the `driverConfig.SequencerStopped` state already is set.

Reply:
hmm, that winds up being a more intrusive change for starting because we need to persist only after validating that the unsafe head is the expected hash, but don't want to report a mismatched head to the user because the sequencer is already running.  So on one hand we want to check the hash matches, then persist state, then check if a change is required and the other we want to check if the change is required, then the hash then persist.

Given that we only change the state if the persistence works, if we also do the persistence on startup as you suggest they'll always be in sync anyway and we shouldn't need to do this.  And if they do somehow get out of sync somehow we can manually edit the file since it's never cached in memory.